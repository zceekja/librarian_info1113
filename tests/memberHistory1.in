member history 100000