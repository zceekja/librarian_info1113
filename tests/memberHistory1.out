user: No members in system.

user: 